core/module.h

```diff
@@ -166,7 +175,11 @@ class Module {
         attr_offsets_(),
         tasks_(),
         igates_(),
-        ogates_() {}
+        ogates_(),
+        active_workers_(),
+        node_constraints_(UNCONSTRAINED_SOCKET),
+        min_allowed_workers_(1),
```

Comment:
In the common case, is the goal of the min to find modules that never got connected?